vue-custom-bootstrap/src/components/RightSideBarLayout.vue

```diff
@@ -1,19 +1,18 @@
 <template>
```

Comment:
When using a component framework, we try to encapsulate all styles inside reusable components. With vue-bootstrap, thre's already a bunch of little helper components we can use.